### PR TITLE
[risk=low][RW-11996] User Apps egress email: add "the" and use correct capitalization

### PR DESCRIPTION
**Pull request by jmthibault79**
Update the egress alert email for RStudio:
* Add "the" before the environment type
* Capitalize RStudio correctly
* Add a period after the DUCC link
* Add spacing

BEFORE

Screenshot 2024-03-04 at 3 24 48 PM

AFTER

Screenshot 2024-03-04 at 3 36 23 PM

Follow up to #8394

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have manually tested this change and my testing process is described above.
- [x] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [x] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [x] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [x] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: api/src/main/resources/emails/egress_remediation/content.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
 <div>This is an automated email from the ${ALL_OF_US} Security Team at Vanderbilt University Medical
   Center. We received an alert related to activity on your account <b>${USERNAME}</b>. This
   alert was triggered because our monitoring system detected potential data egress from
-  your workspace when using <span class="variable"><b>${ENVIRONMENT_TYPE}</b> application.</span><br/>
+  your workspace when using the <span class="variable"><b>${ENVIRONMENT_TYPE}</b> application.</span><br/>
```
Comment (jmthibault79) [inline api/src/main/resources/emails/egress_remediation/content.html:28]: more natural sentence structure

File: api/src/main/java/org/pmiops/workbench/exfiltration/jirahandler/EgressSumologicJiraHandler.java [hunk 2]
```diff
@@ -71,7 +71,9 @@ protected Stream<AtlassianContent> jiraEventDescription(
             ? String.format(
                 "User App name: %s, App type: %s\n",
                 serviceName,
-                appServiceNameToAppType(serviceName).map(AppType::toString).orElse("[unknown]"))
+                appServiceNameToAppType(serviceName)
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/exfiltration/jirahandler/EgressSumologicJiraHandler.java:74]: AppType::toString returns **RSTUDIO** but LeonardoAppUtils::appDisplayName returns **RStudio**

File: api/src/main/java/org/pmiops/workbench/exfiltration/impl/EgressSumologicRemediationService.java [hunk 3]
```diff
@@ -78,7 +79,7 @@ protected void sendEgressRemediationEmail(
     SumologicEgressEvent originalEvent = egressEventMapper.toSumoLogicEvent(event);
     String environmentType =
         appServiceNameToAppType(Strings.nullToEmpty(originalEvent.getSrcGkeServiceName()))
-            .map(AppType::toString)
+            .map(LeonardoAppUtils::appDisplayName)
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/exfiltration/impl/EgressSumologicRemediationService.java:82]: AppType::toString returns **RSTUDIO** but LeonardoAppUtils::appDisplayName returns **RStudio**

**Review by yonghaoy — APPROVED** [review]
nice

**Pull request merged**